### PR TITLE
docs(router): corrected minor typo

### PR DESCRIPTION
**Pull request by m-ahmadi**
just a typo correction

### CODE REVIEW

**Comment by m-ahmadi** [comment]
@googlebot I signed it!

File: packages/router/src/config.ts [hunk 1]
```diff
@@ -420,7 +420,7 @@ export interface Route {
    */
   component?: Type<any>;
   /**
-   * A URL to which to redirect when a the path matches.
+   * A URL to which to redirect when the path matches.
```
Comment (atscott) [inline packages/router/src/config.ts:423]:
```suggestion
   * A URL to redirect to when the path matches.
```
"to which to" appears wrong as well. How about "A URL to redirect to..."?

**Review by atscott — COMMENTED** [review]
Hi @m-ahmadi, thanks for you contribution. Can you also update the commit message to conform to the 100 character requirement in the body? I know it's a little tedious on typo corrections, but this character requirement helps us in many other places to prevent PRs that don't describe well enough why something is changing.

**Comment by atscott** [comment]
Hi @m-ahmadi, looks like CI failed again. Can you rebase using the steps there:

```
          git fetch upstream master;
          git checkout patch-1;
          git rebase upstream/master;
          git push --force-with-lease;
```

**Comment by m-ahmadi** [comment]
> Hi @m-ahmadi, looks like CI failed again. Can you rebase using the steps there:
> 
> ```
>           git fetch upstream master;
>           git checkout patch-1;
>           git rebase upstream/master;
>           git push --force-with-lease;
> ```

Hi @atscott, I tried the steps, but nothing seemed to be happening. it said `Current branch patch-1 is up to date.` after the `git rebase ...` command, and also said `Everything up to date.` after the `git push ...` command.

**Comment by atscott** [comment]
Hi @m-ahmadi, that's alright. I rebased it and pushed the commit. It may have been a difference with the way your git remotes are named. `upstream` is meant to refer to `git@github.com:angular/angular.git`

**Review by atscott — APPROVED** [review]

**Pull request closed without merging**